frame/support/src/traits/storage.rs

```diff
@@ -59,6 +63,25 @@ pub struct StorageInfo {
 	pub max_size: Option<u32>,
 }
 
+impl StorageInfo {
+	pub fn new(pallet: &[u8], storage: &[u8]) -> Self {
```

Comment:
Maybe we can find a better name, the info created this way are quite incomplete, like `new_no_maximum` or `new_unbounded` or `new_no_bound`

But best would be that pallet macro always generate storage info on the pallet, so nobody needs to use this at all. Maybe we should do in a follow-up

Reply:
I think we can just remove this if you enable the generation of storage name and stuff

Reply:
yes removed